sway/commands/force_display_urgency_hint.c

```diff
@@ -11,10 +11,12 @@ struct cmd_results *cmd_force_display_urgency_hint(int argc, char **argv) {
 	char *err;
 	int timeout = (int)strtol(argv[0], &err, 10);
 	if (*err) {
```

Comment:
There are a number of errors not handled here:

* We should error out if `end == argv[0]` (the first character isn't a digit, nothing has been parsed)
* We should set `errno = 0` before the `strtol` call, and error out if `errno != 0` after the call.

Reply:
ahh...right :smile: